src/store/modules/layers.store.js

```diff
@@ -271,11 +271,27 @@ const actions = {
             log.error('Can not remove layer that is not yet added', layerIdOrObject)
         }
     },
-    updateLayer({ commit }, layer) {
-        if (!(layer instanceof AbstractLayer)) {
+    updateLayer({ commit, getters }, layer) {
+        if (layer instanceof AbstractLayer) {
+            commit('updateLayer', layer)
+        } else if (layer instanceof Object && layer.id) {
```

Comment:
That might be good to document the different input possibilities in a JSDoc, especially the "partial update" part where an `id` is required, and then whatever we want to update, in the same JSON object

Reply:
Done